composer.json

```diff
@@ -31,37 +31,43 @@
-        "oat-sa/extension-tao-testqti": "44.21.1.1",
-        "oat-sa/extension-tao-testtaker": "8.9.2",
-        "oat-sa/extension-tao-group": "7.6.2",
-        "oat-sa/extension-tao-item": "11.31.3",
-        "oat-sa/extension-tao-itemqti-pci": "8.3.0",
-        "oat-sa/extension-tao-itemqti-pic": "6.4.0",
-        "oat-sa/extension-tao-test": "15.15.0",
-        "oat-sa/extension-tao-outcome": "13.2.1",
-        "oat-sa/extension-tao-outcomeui": "11.1.7",
-        "oat-sa/extension-tao-outcomerds": "8.2.1",
-        "oat-sa/extension-tao-outcomekeyvalue": "6.1.2",
-        "oat-sa/extension-tao-outcomelti": "5.0.2",
-        "oat-sa/extension-tao-delivery": "15.12.0",
-        "oat-sa/extension-tao-delivery-rdf": "14.16.2",
-        "oat-sa/extension-tao-lti": "15.9.1",
-        "oat-sa/extension-tao-ltideliveryprovider": "12.12.3",
-        "oat-sa/extension-tao-revision": "10.4.0",
-        "oat-sa/extension-tao-mediamanager": "12.31.2.1",
-        "oat-sa/extension-pcisample": "3.6.1",
-        "oat-sa/extension-tao-backoffice": "6.11.2",
-        "oat-sa/extension-tao-proctoring": "20.5.4",
-        "oat-sa/extension-tao-clientdiag": "8.4.4",
-        "oat-sa/extension-tao-eventlog": "3.3.2",
-        "oat-sa/extension-tao-task-queue": "6.6.2",
-        "oat-sa/extension-tao-testqti-previewer": "3.7.2"
+    "oat-sa/generis": "15.20.4",
+    "oat-sa/tao-core": "50.23.0.2",
+    "oat-sa/extension-tao-community": "10.3.0",
+    "oat-sa/extension-tao-funcacl": "7.2.1",
+    "oat-sa/extension-tao-dac-simple": "7.7.2",
+    "oat-sa/extension-tao-itemqti": "29.12.4.2",
+    "oat-sa/extension-tao-testqti": "44.21.1.1",
+    "oat-sa/extension-tao-testtaker": "8.9.2",
+    "oat-sa/extension-tao-group": "7.6.2",
+    "oat-sa/extension-tao-item": "11.31.3",
+    "oat-sa/extension-tao-itemqti-pci": "8.3.0",
+    "oat-sa/extension-tao-itemqti-pic": "6.4.0",
+    "oat-sa/extension-tao-test": "15.15.0",
+    "oat-sa/extension-tao-outcome": "13.2.1",
+    "oat-sa/extension-tao-outcomeui": "11.1.7",
+    "oat-sa/extension-tao-outcomerds": "8.2.1",
+    "oat-sa/extension-tao-outcomekeyvalue": "6.1.2",
+    "oat-sa/extension-tao-outcomelti": "5.0.2",
+    "oat-sa/extension-tao-delivery": "15.12.0",
+    "oat-sa/extension-tao-delivery-rdf": "14.16.2",
+    "oat-sa/extension-tao-lti": "15.9.1",
+    "oat-sa/extension-tao-ltideliveryprovider": "12.12.3",
+    "oat-sa/extension-tao-revision": "10.4.0",
+    "oat-sa/extension-tao-mediamanager": "12.31.2.1",
+    "oat-sa/extension-pcisample": "3.6.1",
+    "oat-sa/extension-tao-backoffice": "6.11.2",
+    "oat-sa/extension-tao-proctoring": "20.5.4",
+    "oat-sa/extension-tao-clientdiag": "8.4.4",
+    "oat-sa/extension-tao-eventlog": "3.3.2",
+    "oat-sa/extension-tao-task-queue": "6.6.2",
+    "oat-sa/extension-tao-testqti-previewer": "3.7.2"
+  },
+  "require-dev": {
+    "mikey179/vfsstream": "~1",
+    "nimut/phpunit-merger": "~1",
+    "phpspec/prophecy": "~1",
+    "phpunit/phpunit": "^9.3"
```

Comment:
Will this be compatible with all extensions PHP unit?

Reply:
They all already run under ~9

Reply:
Darn, generis is still stuck on 8.5 indeed.
Extended the dependencies.

We'll have to update them sooner or later.